exchanges/alphapoint/alphapoint.go

```diff
@@ -51,6 +51,7 @@ type Alphapoint struct {
 // SetDefaults sets current default settings
 func (a *Alphapoint) SetDefaults() {
 	a.APIUrl = alphapointDefaultAPIURL
+	a.HTTPDebugging = false
```

Comment:
This will be false by default

Reply:
Thanks!
I have removed the setting of default value for HTTPDebugging across all exchanges